contributing-guides/maintainers-guide.md

```diff
@@ -56,22 +56,20 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**, and
-  (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (2) have the **review comments addressed**,
+  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
```

Comment:
```suggestion
  (3) are **approved by two maintainers**
      (the second maintainer can merge immediately after approving)
```

Reply:
I agree with `get approved reviews` → `are approved`, but I don't understand the newline.

Reply:
better readability. On my screen at least I cannot see the whole line in your proposal.
https://sembr.org